app/views/registrations/school_not_confirmed/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% content_for :before_content, govuk_back_link(
+      text: "Back",
+      href: registrations_school_profile_path)
+    %>
+    <h2 class="govuk-heading-l">Your school needs to complete registration</h2>
+    <p>You or one of your colleagues may have tried to register before and need verify your email.</p>
+    <p>This email will be used to sign in so check with your colleagues if you need access.</p>
+    <p>What you can do</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>check your inbox for an email from <span>TWD Early Careers Framework</span> (it may be in your spam or junk folder)</li>
+      <li>or your colleague can verify, using the link sent to their email.</li>
+      <li>if the link has expired, you can <%= govuk_link_to "register", registrations_question_one_path %> again.</li>
```

Comment:
A bit weird - two of the options end with a `.`, and all start with a lowercase, hmm. 

Then again, I imagine it came from designs, so not sure if we want to pursue changing that.

Reply:
It did come from the designs, that's right. I'll speak to Shani about it. Personally, I don't have any strong feelings about it.